macros/provider_test.go

```diff
@@ -0,0 +1,38 @@
+package macros
+
+import (
+	"testing"
+
+	"github.com/prebid/prebid-server/exchange/entities"
+)
+
+func Test_macroProvider_GetMacro(t *testing.T) {
+	type args struct {
+		keys []string
+	}
+	tests := []struct {
+		name string
+		args args
+		want map[string]string
+	}{
+		{
+			name: "get all macros success",
+			args: args{keys: []string{MacroKeyBidID, MacroKeyAccountID, MacroKeyAppBundle, MacroKeyPubDomain,
```

Comment:
IMHO, this structure for the test cases kinda seem odd to me. It's like you're creating test cases inside of test cases with `args`. I'd rather suggest we create a separate test case for each macro. That way it's easier to read and understand each test case

Reply:
Added unit test for each macro.